src/js/module/map/map.js

```diff
@@ -23,33 +31,45 @@ const MapView = View.extend({
         });
         this.panorama = this.map.getStreetView();
 
-        google.maps.event.addListener(this.panorama, 'visible_changed', function() {
-          this.app.trigger(Const.Event.VISIBLE_CHANGED, this.panorama.getVisible());
-        }.bind(this));
-      }.bind(this));
-
+        mapEvent(google.maps.event, this.panorama)
+          .addListener('visible_changed', () => {
```

Comment:
Those events are thrown by google maps. I created small wrapper to chain them, so we dont need to type everytime
```javascript
google.maps.event.addListener(this.panorama, 'event-name', callback);
google.maps.event.addListener(this.panorama, 'event-name-2', callback);
```
instead we can register events this way:
```javascript
mapEvent(google,maps.event, this.panorama)
  .addListener('event-name', callback)
  .addListener('event-name-2', callback);
```